code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -107,12 +120,14 @@
 		if(insert(AM) == -1) // limit reached
 			break
 
-/obj/structure/closet/proc/open()
-	if(opened || !can_open())
+/obj/structure/closet/proc/open(mob/living/user)
+	if(opened || !can_open(user))
 		return
 	playsound(loc, open_sound, 15, 1, -3)
 	opened = 1
-	density = 0
+	if(!dense_when_open)
+		density = 0
+	climb_time *= 0.5 //it's faster to climb onto an open thing

```

Comment:
Technically it's into, given that you can't climb onto lockers.
Make it a var, so open crate-climbing is faster but theoretical locker-climbing is slower? On the other hand it doesn't make sense to climb onto an open locker because it wouldn't be dense and that defeats the point of climbing it in the first place